src/lib/blueprint.ts

```diff
@@ -2,14 +2,65 @@ import type { Openapi } from './openapi.js'
 
 export interface Blueprint {
   name: string
+  routes: Route[]
 }
 
+interface Route {
+  name: string
+  path: string
+  description: string
+  namespace: Namespace | null
+  endpoints: Endpoint[]
+  subroutes: Route[]
+}
+
+interface Namespace {
+  name: string
+  path: string
+}
```

Comment:
It's not clear to me whether we want to keep Namespace. It's usefulness is not clear to me but maybe I'm misunderstanding something

Reply:
We should keep namespace, even though it only applies to ACS at the moment. I think the usefulness will be more clear when trying to use the blueprint for sdk gen.

I'm not sure how to encode the namespace exists inside the openapi spec. We may need to remove `name` for now and say a namespace is something that only contains routes.

So logic is something like find all namespaces, then build all route

```
if namespace if path contains no endpoints
is route if path contains at least one endpoint
a route has namespace if route is under the path of a namespace
```